generatedComponentConfig.js

```diff
@@ -66,10 +66,10 @@ import DefaultLabelValueView from './packages/terra-clinical-label-value-view/ex
 import MultipleValueLabelValueView from './packages/terra-clinical-label-value-view/examples/test-examples/MultipleValueLabelValueView.example';
 import NodeValueLabelValueView from './packages/terra-clinical-label-value-view/examples/test-examples/NodeValueLabelValueView.example';
 import TextValueLabelValueView from './packages/terra-clinical-label-value-view/examples/test-examples/TextValueLabelValueView.example';
-import ContentNoDataView from './packages/terra-clinical-no-data-view/tests/nightwatch/ContentNoDataView';
-import DefaultNoDataView from './packages/terra-clinical-no-data-view/tests/nightwatch/DefaultNoDataView';
-import HiddenNoDataView from './packages/terra-clinical-no-data-view/tests/nightwatch/HiddenNoDataView';
-import TextNoDataView from './packages/terra-clinical-no-data-view/tests/nightwatch/TextNoDataView';
+import ContentNoDataView from './packages/terra-clinical-no-data-view/tests/wdio/ContentNoDataView';
```

Comment:
Again, I know you already did this work, but I don't think we should move these file either. See conversation here #296

Reply:
https://github.com/cerner/terra-clinical/pull/309/commits/7891fe0d9d2e3e0fce9280e2f450124d39ee5463